docs/api-types.md

```diff
@@ -0,0 +1,100 @@
+# API types documentation
+
+This document describes format of objects used in various API's across the project.
```

Comment:
why do you need a tick in `API's`?

Reply:
Dunno, it seems more readable than `APIs`.
I guess you don't think so, I'll remove.